dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java

```diff
@@ -47,10 +45,6 @@
 public abstract class GA4GHIT {
     protected static final DropwizardTestSupport<DockstoreWebserviceConfiguration> SUPPORT = new DropwizardTestSupport<>(
             DockstoreWebserviceApplication.class, CommonTestUtilities.CONFIG_PATH);
-    static final ObjectMapper MAPPER = Jackson.newObjectMapper();
```

Comment:
Can't this be changed to static final ObjectMapper MAPPER = SUPPORT.getObjectMapper(); and not require changes in the other files?

Reply:
I thought so, but I got an `ExceptionInInitializerError` from within DropwizardTestSupport either setting it that way or in @BeforeClass.